src/utils/hierarchy.ts

```diff
@@ -52,16 +52,23 @@ export function getColumnWidth(
     const columnKey = showedFields?.indexOf(currentField)
     const currentColumnShift = currentField?.hierarchyShift
     const nextColumnShift = showedFields[columnKey + 1]?.hierarchyShift
+    // Ability to set last column width if one of previous columns have width = 0
+    const notFixedWidthColumn = showedFields.find(item => item.width === 0)
```

Comment:
Judging by the neighbour usage `showedFields` can and will be undefined at some point

Reply:
Also probably `noFixedWidthColumn`, as it's checking an existance of any fixed columns, not if specific one match this condition